base/array.jl

```diff
@@ -791,9 +791,9 @@ function find(testf::Function, A::AbstractArray)
     I
 end
 
-function find(A::AbstractArray)
+function find(A)
     nnzA = countnz(A)
-    I = similar(A, Int, nnzA)
+    I = Vector{Int}(nnzA)
```

Comment:
I think `find` always returns a vector: this is what `similar` did as a single dimension argument was passed. Anyway it doesn't make sense to keep the size of the input array.